bin/test_benchmarks

```diff
@@ -53,4 +57,43 @@ pushd test
     "
 popd || true
 
+# gather auto benchmark data for website
+
+# container size
+container_size=$(docker images \
+  | grep "cyberark\/secretless-broker[^-]" \
+  | awk '{ print $7 }')
```

Comment:
```
docker images -q --filter=reference='cyberark/secretless-broker' --format="{{ .Size }}"
```

Ideally we should be filtering by LABEL but we don't have one set in the final Dockerfile